weatherApp/Features/WeatherList/UI/CurrentWeatherListPresenter.swift

```diff
@@ -35,4 +42,9 @@ class CurrentWeatherListPresenter {
         return currentWeatherList.at(index)
     }
     
+    
+    func handlePushDetailWeatherViewController(currentWeather: CurrentWeatherViewModel) {
```

Comment:
I would name this function something like **handleSelectedCity**. View controller should not be aware of what will happen on city item selection.